src/apps/watchfaces/watchface_dual_time.cpp

```diff
@@ -0,0 +1,294 @@
+#include "./apps/watchfaces/watchface_dual_time.h"
+
+#include <config.h>
+#include <gfx_util.h>
+#include <osw_app.h>
+#include <osw_config_keys.h>
+#include <osw_hal.h>
+#include <time.h>
+
+#include "./apps/watchfaces/watchface.h"
+#include "bma400_defs.h"
+
+#define COLOR_BLACK rgb565(0, 0, 0)
+
+#define BUF_W 240
+#define BUF_H 240
+
+void stepsDisplay() {
+#ifdef OSW_FEATURE_STATS_STEPS
+  uint8_t w = 8;
+  OswAppWatchface::drawStepHistory(OswUI::getInstance(), (DISP_W / 2) - w * 3.5, 180, w, w * 4,
+                                   OswConfigAllKeys::stepsPerDay.get());
+#else
+  OswHal* hal = OswHal::getInstance();
+  uint32_t steps = hal->environment->getStepsToday();
+  hal->gfx()->setTextCenterAligned();
+  hal->gfx()->setTextSize(2);
+  hal->gfx()->setTextCursor(120, 210 - hal->gfx()->getTextOfsetRows(1) / 2);
+
+  hal->gfx()->print(steps);
+#endif
+}
+
+void localDateDisplay(const uint8_t& showDateFormat, uint8_t startCoord) {
+  uint32_t dayInt = 0;
+  uint32_t monthInt = 0;
+  uint32_t yearInt = 0;
+  OswHal* hal = OswHal::getInstance();
+  const char* weekday = hal->getWeekday();
+
+  hal->getDate(&dayInt, &monthInt, &yearInt);
+
+  // we want to output a value like "Wed, 05/02/2021"
```

Comment:
This is also part of the current digital watchface, please reuse its code for that instead!